lib/validation.js

```diff
@@ -7,23 +7,36 @@ const ajv = new Ajv({ coerceTypes: true })
 
 const bodySchema = Symbol('body-schema')
 const querystringSchema = Symbol('querystring-schema')
-const outputSchema = Symbol('output-schema')
 const paramsSchema = Symbol('params-schema')
+const responseSchema = Symbol('response-schema')
 
 const schemas = require('./schemas.json')
 const inputSchemaError = fastJsonStringify(schemas.inputSchemaError)
 
+function getValidatorForStatusCodeSchema (statusCodeDefinition) {
+  return fastJsonStringify(statusCodeDefinition)
+}
+
+function getResponseSchema (responseSchemaDefinition) {
+  var statusCodes = Object.keys(responseSchemaDefinition)
+  return statusCodes.reduce(function (r, statusCode) {
+    r[statusCode] = getValidatorForStatusCodeSchema(responseSchemaDefinition[statusCode])
+    return r
+  }, {})
+}
+
 function build (opts) {
   if (!opts.schema) {
-    opts[outputSchema] = fastSafeStringify
+    opts[responseSchema] = {default: fastSafeStringify}
```

Comment:
there is no need to configure a default for `fastSafeStringify` use in place of the `out[responseSchema].default()` call.